app/views/users/_answered.html.erb

```diff
@@ -37,10 +37,12 @@
   </div>
 </div>
 
-<div class="text-center">
-<% if @pagy %>
-  <%== pagy_bootstrap_nav @pagy %>
-<% else %>
-  <%= will_paginate @questions, :renderer => WillPaginate::ActionView::BootstrapLinkRenderer unless @unpaginated || @questions.empty? %>
+<% if @answered_questions.present? %>
```

Comment:
So, is the issue here that one of the pagination elements is for answered questions and the other is for unanswered? Looking at the two includes at 

https://github.com/publiclab/plots2/blob/063352142013d88b1b76bf1b4150fac9de1b5f9f/app/views/tag/show/_tab_content.html.erb#L28-L37

To test this theory, we could try commenting out one and seeing if the double pagination helper disappears.

Just to note, I believe we are OK eliminating the distinction between answered and unanswered in this display. Pinging @ebarry to confirm though.

Reply:
yes it does sound ok to eliminate the "answered | unanswered" toggle and the associated pagination. 

One idea towards helping people choose which question card to potentially click on is to pull out the # of comments that is contained in the `...` on each question card, and perhaps show these #s directly on the card to give people a sense of where there's a lot of dialogue/engagement.

Reply:
OK, great, so confirmed we can just remove the toggle, and hopefully that resolves the double pagination too!

I'm breaking out the design discussion portion into its own issue at: https://github.com/publiclab/plots2/issues/8731

Reply:
Also we will move over to https://github.com/publiclab/plots2/pull/8689 -- thanks!